### PR TITLE
Add Heatmap Example

### PR DESCRIPTION
**Pull request by barakplasma**
Screenshots:
screen shot 2019-02-11 at 11 27 41
screen shot 2019-02-11 at 11 28 25

This PR adds a heatmap example to the project.
Complements my PR to improve documentation for the feature: https://github.com/google-map-react/google-map-react/pull/716

### CODE REVIEW

**Comment by itsmichaeldiego** [comment]
@barakplasma Hey! Thank you so much, It is looking good! Just be careful you added `package-lock.json` and here we're using `yarn` instead., if you could fix that and tell me I will merge this asap.

**Comment by barakplasma** [comment]
@itsmichaeldiego I deleted the package-lock.json . Do you want me to update the yarn.lock?

File: src/index.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Home from './Home';
 import Main from './examples/Main';
 import SearchBox from './examples/Searchbox';
 import Autocomplete from './examples/Autocomplete';
+import Heatmap from './examples/Heatmap';
```
Comment (itsmichaeldiego) [inline src/index.js]: Can we align it top? Sorry, OCD 😂

File: src/index.js [hunk 2]
```diff
@@ -28,6 +29,7 @@ ReactDOM.render(
         <Route path={`${defaultPath}default`} component={Main} />
         <Route path={`${defaultPath}searchbox`} component={SearchBox} />
         <Route path={`${defaultPath}autocomplete`} component={Autocomplete} />
+        <Route path={`${defaultPath}heatmap`} component={Heatmap} />
```
Comment (itsmichaeldiego) [inline src/index.js]: Same as above

File: src/index.js (hunk 1 above)
Comment (itsmichaeldiego) [inline src/index.js]: Can we align it like the rest? Sorry, OCD 😂

File: src/examples/Heatmap.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      },
+    };
+
+    return (
+      <Fragment>
+        {!isEmpty(places) && (
+          <GoogleMap
+            defaultZoom={10}
+            defaultCenter={LOS_ANGELES_CENTER}
+            heatmapLibrary
```
Comment (itsmichaeldiego) [inline src/examples/Heatmap.js]: Can we use instead:
```
    bootstrapURLKeys={{
    key: 'xxxxxxxxxxxxxx',
    libraries: ['visualization'],
  }}
```

Due to https://github.com/google-map-react/google-map-react/issues/673 ?

**Comment by itsmichaeldiego** [comment]
@barakplasma Hey man! First let me say this is looking great now, I just left two really minor comments and one where I want to hear what you think: https://github.com/google-map-react/google-map-react-examples/pull/19#discussion_r256865107

File: src/index.js (hunk 1 above)
Comment (barakplasma) [reply src/index.js]: not sure if I understood what you meant by align, so I sorted the import lines by length in https://github.com/google-map-react/google-map-react-examples/pull/19/commits/d9ff68f7d25dfd4e3e9bb3ce991aa7edfc39f357

File: src/index.js (hunk 2 above)
Comment (barakplasma) [reply src/index.js]: not sure if I understood what you meant by align, so I sorted the import lines by length in https://github.com/google-map-react/google-map-react-examples/pull/19/commits/d9ff68f7d25dfd4e3e9bb3ce991aa7edfc39f357

File: src/examples/Heatmap.js (hunk 3 above)
Comment (barakplasma) [reply src/examples/Heatmap.js]: done in 8eab280

**Comment by barakplasma** [comment]
I'll correct the line breaks soon in order to pass CI

**Comment by barakplasma** [comment]
@itsmichaeldiego sorry for the delay. I changed the line endings in order to pass eslint. This PR and https://github.com/google-map-react/google-map-react/pull/716#discussion_r256863681 seem ready to me for merge.

**Comment by barakplasma** [comment]
Thanks @itsmichaeldiego !

I just want to remind you to run 
```sh
$ npm run deploy
```
from the master branch in order to deploy the Heatmap Example to the gh-pages site. Currently, the demo doesn't have the heatmap example.

Alternatively, I can make another PR with that change for the gh-pages branch.

**Comment by itsmichaeldiego** [comment]
@barakplasma Yeah I did not have time to publish, its published now! Let me know if it works for you

**Comment by itsmichaeldiego** [comment]
@barakplasma BTW you might want to create a PR to add the example into the README list: https://github.com/google-map-react/google-map-react-examples#examples

**Pull request merged**